miasm2/expression/expression.py

```diff
@@ -69,6 +69,14 @@ def wrapped(expr, callback, test_visit=lambda x: True):
     return wrapped
 
 
+DISPLAYED_CHARS = set('abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0X123456789_')
+
+def is_prefixed_opname(name):
```

Comment:
From what the function seems to do, the name is quite misleading. It is more about the charset used than a notion of prefix/suffix, no?